signing_service/signing_service/signing_service.py

```diff
@@ -268,6 +273,25 @@ def _handle_connection(
             # If received frame is correct, validate transaction.
             else:
                 golem_message_response = self._get_signed_transaction(middleman_message.payload)
+                if isinstance(golem_message_response, SignedTransaction):
+                    transaction_sum_so_far = self._get_signing_service_daily_transaction_sum_so_far()
```

Comment:
How about changing it to work this way:
1. You read value only once, on `__init__` (and maybe when connection is dropped and SS is trying to reconnect too?).
2. You keep the value in class attribute like `self.signing_service_daily_transaction_sum_so_far`.
3. You only update this value and write to file.

This way we will decrease IO operations by half, and there should not be much risk of those two values to become different.

@dybi What do you think?

Reply:
Done